jaeger/cmd/install.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/linkerd/linkerd2/jaeger/flag"
+	jaeger "github.com/linkerd/linkerd2/jaeger/values"
+	"github.com/linkerd/linkerd2/pkg/charts"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/helm/pkg/chartutil"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	templatesJaeger = []string{
+		"templates/namespace.yaml",
+		"templates/tracing-rbac.yaml",
+		"templates/tracing.yaml",
+	}
+)
+
+func newCmdInstall() *cobra.Command {
+	var skipChecks bool
+
+	values, err := jaeger.NewValues()
+	if err != nil {
+		fmt.Fprint(os.Stderr, err.Error())
+		os.Exit(1)
+	}
+
+	flags, installOnlyFlagSet := makeInstallFlags(values)
```

Comment:
I think we can ditch all of this fancy flag machinery in favor of just plain cobra flags.